BackEnd/src/db/games.ts

```diff
@@ -408,3 +408,9 @@ export async function GetDbPlayerGamesBySeasonId(seasonId: string): Promise<Play
   }
   return await PlayerGameModel.find(filter);
 }
+
+export async function GetDbGamesBySeasonId(seasonId: number ): Promise<PlayerGameModel[]> {
```

Comment:
Can this be renamed to `GetDbPlayerGamesBySeasonId`. As there is also a GameModel.

Reply:
That name already exists, ive deleted this function and instead used the existing function

Reply:
Ok perfect.